docs/content/dagster-cloud/managing-deployments/managing-multiple-projects-and-teams.mdx

```diff
@@ -0,0 +1,433 @@
+
+#### Example with GitHub CI/CD on Hybrid deployment
+
+1. **For each repository**, use the CI/CD workflow provided in [Dagster Cloud Hybrid quickstart repository](https://github.com/dagster-io/dagster-cloud-hybrid-quickstart/blob/main/.github/workflows/dagster-cloud-deploy.yml).
+
+2. **For each project in the repository**, configure a code location in the [`dagster_cloud.yaml` file](/dagster-cloud/managing-deployments/dagster-cloud-yaml):
+
+   ```yaml
+   # dagster_cloud.yml
+
+   locations:
+     - location_name: project_a
+       code_source:
+         package_name: project_a
+       build:
+         # ...
+     - location_name: project_b
+       code_source:
+         package_name: project_b
+       build:
+         # ...
+   ```
+
+3. In the repository's `dagster-cloud-deploy.yml` file, modify the CI/CD workflow to deploy all code locations for the repository:
+
+   ```yaml
+   # .github/workflows/dagster-cloud-deploy.yml
+
+   jobs:
+     dagster-cloud-deploy:
+       # ...
+       steps:
+         - name: Update build session with image tag for "project_a" code location
+           id: ci-set-build-output-project-a
+           if: steps.prerun.outputs.result != 'skip'
+           uses: dagster-io/dagster-cloud-action/actions/utils/dagster-cloud-cli@v0.1
+           with:
+             command: "ci set-build-output --location-name=project_a --image-tag=$IMAGE_TAG"
+
+         - name: Update build session with image tag for "project_b" code location
+           id: ci-set-build-output-project-b
+           if: steps.prerun.outputs.result != 'skip'
+           uses: dagster-io/dagster-cloud-action/actions/utils/dagster-cloud-cli@v0.1
+           with:
+             command: "ci set-build-output --location-name=project_b --image-tag=$IMAGE_TAG"
+         # ...
+   ```
+
+---
+
+## Isolating execution context between projects
+
+Separating execution context between projects can have several motivations:
+
+- Facilitating separation of duty between teams to prevent access to sensitive data
+- Differing compute environments and requirements, such as different architecture, cloud provider, etc.
+- Reducing impact on other projects. For example, a project with a large number of runs can impact the performance of other projects.
+
+In order from least to most isolated, there are three levels of isolation:
+
+- [Code location](#code-location-isolation)
+- [Agent](#agent-isolation)
+- [Deployment](#deployment-isolation)
+
+### Code location isolation
+
+If you have no specific requirements for isolation beyond the ability to deploy and run multiple projects, you can use a single agent and deployment to manage all your projects as individual code locations.
+
+<Image
+alt="Diagram of isolation at the code location level"
+src="/images/dagster-cloud/managing-deployments/isolation-level-code-locations.png"
+width={1167}
+height={384}
+/>
+
+<table
+  className="table"
+  style={{
+    width: "100%",
+  }}
+>
+  <thead>
```

Comment:
Like the code bases section, I put the **Pros** and **Cons** into a similar table.

We could also add a section that rolls up all the pros and cons of each isolation approach, which would allow readers to compare them all. I can do that in a follow-up PR if it's of interest.